python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -108,6 +108,36 @@ def isPseudoSource(source, pseudoColumns):
     return isPseudo
 
 
+def computePsfKernelImage(mExposure, psfCenter):
+    """Compute the PSF kernel image and update the multiband exposure
+    if not all of the PSF images could be computed.
+
+    Parameters
+    ----------
+    psfCenter : `tuple` or `Point2I` or `Point2D`
+        The location `(x, y)` used as the center of the PSF.
+
+    Returns
+    -------
+    psfModels : `np.ndarray`
+        The multiband PSF image
+    mExposure : `MultibandExposure`
+        The exposure, updated to only use bands that
+        successfully generated a PSF image.
+    """
+    if not isinstance(psfCenter, geom.Point2D):
+        psfCenter = geom.Point2D(*psfCenter)
+
+    try:
+        psfModels = mExposure.computePsfKernelImage(psfCenter)
+    except IncompleteDataError as e:
```

Comment:
I think this is a good example of a case where it is easier to ask forgiveness than acceptance. I originally used an `allowMissing` parameter but it requires an additional check by the user to see if any bands were missing. Since this check is already being done while creating the PSF image I wanted to find a way to avoid having to validate the output again, and this seemed like a good way to ensure that users who don't expect a bad PSF will get an error, and users who know that it might happen do not have to check for it.

Reply:
Ok, sounds reasonable to me.